Sources/Kitura/RouterResponse.swift

```diff
@@ -179,6 +179,143 @@ public class RouterResponse {
         }
     }
 
+    /// Describes an optional attribute of a cookie.
+    public struct CookieAttribute {
```

Comment:
I'm sorry, I don't see why `struct CookieAttribute` belongs as nested inside `RouterResponse`. Can you explain the rationale for that?

Reply:
As cookies are closely related with HTTP response, we thought of adding `struct CookieAttribute` inside `RouterResponse`. If you think we should keep this strust independent, I can do that.

Reply:
@djones6 @pushkarnk what do you think?

Reply:
Hmm. Though I was involved in the decision of making this a nested type,  I'm now thinking of how the name of the type sounds. `RouterResponse.CookieAttribute` doesn't sound logical ( like `RouterResponse.Cookie` or `Cookie.CookieAttribute` would have).

Reply:
I agree, though I suspect inventing a top level type called 'Cookie' to nest it in would be asking for trouble.

Could we make a public extension to HTTPCookie or is that bad practice?

Reply:
Bad practice I think.